### ISSUE TITLE
Repetitive group not returning correct number of symbols

### ISSUE THREAD

**Issue by raunakkathuria** [binary-com/perl-Protocol-FIX#16]
Issue #13 fixed the parsing for message with repetitive group. 
There is one more slight issue, when we parse the messge, the number of symbols returned does not match the one in the message.

The following script output only `USD` instead of both `EUR`, and `USD`

```
use strict;
use warnings;

use Data::Dump qw(dump);
use Protocol::FIX;

my $protocol = Protocol::FIX->new('FIX44');

my $market_data_message =
    "8=FIX.4.4\0019=126\00135=V\001262=KaOjuV\001265=1\00134=1\00149=Client1\00152=20210304-03:49:19.636\001263=1\00156=FixServer\001264=1\001146=2\00155=EUR\00155=USD\001267=2\001269=0\001269=1\00110=232\001\n";

my ($message, $error) = $protocol->parse_message(\$market_data_message);

warn dump($message);

my $symbols = $message->value('InstrmtMDReqGrp')->value('NoRelatedSym');

my @requested_symbols = ();
foreach my $symbol (@$symbols) {
    push @requested_symbols, $symbol->value('Instrument')->value('Symbol');
}

warn dump(@requested_symbols); # Dumps only USD and not EUR and USD

1;
```

**Comment by raunakkathuria** [comment]
@basiliscos any chance if you can have a look?

**Comment by raunakkathuria** [comment]
Fixed in #17
